cartographer/mapping_3d/local_trajectory_builder.cc

```diff
@@ -224,7 +226,7 @@ LocalTrajectoryBuilder::InsertIntoSubmap(
               high_resolution_point_cloud,
               low_resolution_point_cloud,
               rotational_scan_matcher_histogram,
-              pose_observation}),
+              pose_estimate}),
```

Comment:
For the future: Changes like these can easily be done in a separate PR. This one is already large enough.